src/main/java/carracing/domain/Car.java

```diff
@@ -0,0 +1,29 @@
+package carracing.domain;
+
+public class Car {
+    private String location;
+    private static final int MOVE_NUMBER = 4;
```

Comment:
```suggestion
    private static final int MOVE_NUMBER = 4;
    
    private String location;
```

클래스 변수가 인스턴수 변수 위에 존재하는 것이 관례입니다!
또한 사이에 공백을  추가한다면 가독성이 상승합니다 :)

Reply:
앗 감사합니다 :)
관련해서 여쭤보고 싶은 것이 있는데 클래스 변수끼리와 인스턴스 변수끼리의 공백은 혹시 어떻게 처리하시는 편인가요?
항상 모든걸 다띄우기에는 오히려 너무 가독성 중시라 코드 라인이 너무 길어지는 경향이 있는것 같아서요 😢